jujugui/static/gui/src/app/app.js

```diff
@@ -1527,9 +1527,7 @@ YUI.add('juju-gui', function(Y) {
     _renderProviderLogo: function() {
       const container = document.getElementById('provider-logo-container');
       const cloudProvider = this.env.get('providerType');
-      const providerDetails = cloudProvider ?
-        views.utils.getCloudProviderDetails(cloudProvider) :
-        {};
+      let providerDetails = views.utils.getCloudProviderDetails(cloudProvider);
```

Comment:
So the old code only called getCloudProviderDetails if there was a cloudProvider value and now it's calling it all the time. Is this safe? Is this wasted cycles?